fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -471,6 +552,7 @@ private void dispatch() throws IOException {
                 handleQuit();
                 break;
             case COM_QUERY:
+            case COM_STMT_PREPARE:
                 handleQuery();
```

Comment:
Is this PR support JDBC `useServerPrepStmts=true` option?

Reply:
yes, using sql text will bring some problems like sql injection,   in addition,  prepare statement here will skip parser. In future we also can use it skip parser or planner, but the work will be more. so util now we only offer 2 benefits